examples/fabric/nested_ingestion_pipeline/README.md

```diff
@@ -0,0 +1,96 @@
+# Nested pipelines
+
+The following example, exemplifies the utilization of nested pipelines including control activities like ForEach and If.
+
+![Nested Pipelines](pl_main_pipeline.png)
+
+The main pipeline is composed of 2 top activities:
+
+- a lookup activity named "Read Configuration File
+- a ForEach activity names "ForEach"
```

Comment:
```suggestion
- a ForEach activity named "ForEach"
```

Reply:
Typo corrected and I also I've renamed the ForEach activity to "ForEachYearMonthPair" as per your suggestion on the other comment as well.